### ISSUE TITLE
PDF not rendering with expo SDK 35.0.0 in iOS

### ISSUE THREAD

**Issue by SaravanakumarSubramanian** [xcarpentier/rn-pdf-reader-js#80]
PDF not rendering with expo SDK 35.0.0 in iOS,
displays loader indefinitely but same works in android though

**Comment by veleda3** [comment]
have the same issue, does not work on SDK 35 on iOS

**Comment by sguha00** [comment]
I was able to get it to work with loading the package from commit `28ae48fbf0803bac0dc02170d78c4b229cf2d7d9`

e.g. in package.json:
"rn-pdf-reader-js": "xcarpentier/rn-pdf-reader-js#28ae48fbf0803bac0dc02170d78c4b229cf2d7d9",

**Comment by sguha00** [comment]
The above works for me in ios, but now Android crashes when loading a pdf

**Comment by pavawit** [comment]
Same here.
It's about expo-file-system.
It works again when I backward expo version to 33.0.0 which The old way to import expo-file-system is compatible.

**Comment by wardjk87** [comment]
I think it may be react-native-webview. See following issue: https://github.com/react-native-community/react-native-webview/issues/936

**Comment by oschvr** [comment]
Anyone got a workaround?

**Comment by wardjk87** [comment]
The code I posted above works. For iOS, you can just import and utilize react-native-webview and it works.

**Comment by oschvr** [comment]
@wardjk87 If using Expo 35.0, with your solution, it throws an Invariant Violation. "RNCWebViewer" is not defined or found in the UI Manager.

<img width="205" alt="os 2019-11-13 at 10 44 16" src="https://user-images.githubusercontent.com/5449175/68756796-b5526a80-0602-11ea-9c15-6f93a03268a0.png">

See this -> https://github.com/react-native-community/react-native-webview/issues/832#issuecomment-527190628

**Comment by punisher97** [comment]
I working WebView on iOS 13 with useWebKit:
```
 <WebView
        originWhitelist={["http://", "https://", "file://", "data:"]}
        useWebKit={true}
        source={{ uri: pdf }}
      />
```

**Comment by punisher97** [comment]
Hi, I had same issue, the solution it is enable useWebKit in iOS webview:

```js
...
  render() {
    const { ready, data, ios, android } = this.state
    const { style, webviewStyle, onLoad, noLoader, onLoadEnd, onError  } = this.props

    if (data && ios) {
      return (
        <View style={[styles.container, style]}>
          {!noLoader && !ready && <Loader />}
          <WebView
            onLoad={()=>{
              this.setState({ready: true});
              if(onLoad) {
                onLoad();
              }
            }}
            originWhitelist={['http://*', 'https://*', 'file://*', 'data:*']}
            style={[styles.webview, webviewStyle]}
            source={{ uri: data }}
            useWebKit={true}
          />
        </View>
      )
    }
...
```

**Comment by ALEEF02** [comment]
Also having this issue on iOS ever since updating to expo 35.0.1 from 34.0.2 today. Hoping for a fix soon! I might revert back to 34.0.2 in the meantime. I am on react-native-webview version 7.5.1.
![IMG_4637](https://user-images.githubusercontent.com/12904259/69080740-6127fb80-09f1-11ea-8bc6-32de6e1a9636.PNG)

**Comment by danigonlinea** [comment]
My problem is that I receive a pdf in base64 format and also with the newest version of IOs, it stops working. I made a little trick which make that work again, using **Webview** from _react-native-webview_ which consist into parse this base64 pdf into the internal storage and this URI that it is the result of that, can be passed to the Webview which it understands.

```
const [file, setFile] = useState(false);

useEffect(() => {
    if (Platform.OS === 'ios') {
      FileSystem.downloadAsync(base64, `${FileSystem.documentDirectory}temporalPdfFile.pdf`)
        .then(({ uri }) => {
          setFile(uri);
        })
        .catch(error => {
          console.error(error);
        });
    }
  }, [base64]);

  if (extension === 'pdf') {
    if (Platform.OS === 'ios') {
      return file ? (
        <WebView
          originWhitelist={['*']}
          style={{ flex: 1, backgroundColor: '#FFF' }}
          source={{
            uri: file,
          }}
        />
      ) : (
        <ActivityIndicator size="large" style={{ marginVertical: 8 }} />
      );
    }
    return (
      <ContainerPDF>
        <PDFReader style={{ paddingTop: 0 }} source={{ base64 }} />
      </ContainerPDF>
    );
```
I can say this is working for Expo 35.0.0 and IOs 13.

**Comment by xcarpentier** [comment]
Should be fixed at `v0.5.0`.
